src/components/Molecules/List/TweetList/TweetList.tsx

```diff
@@ -1,32 +1,66 @@
 import React from 'react';
-import cx from 'classnames';
-import Tweet from '../../Card/Tweet';
+import { withStyles } from '@material-ui/core/styles';
+import {
+  Card,
+  CardActionArea,
+  CardContent,
+  Typography,
+  CardActions,
+  Button,
+} from '@material-ui/core';
 
 interface ITweetListProps {
   tweets: any[];
-  className?: string;
 }
 
-const TweetList = ({ tweets, className = '' }: ITweetListProps) => {
-  if (tweets.length > 0) {
-    return (
-      <div className={cx(className)}>
-        {tweets.map((tweet, index) => (
-          <Tweet
-            key={index}
-            name={tweet.user.name}
-            screenName={tweet.user.screen_name}
-            createdAt={tweet.created_at}
-            text={tweet.text}
-            favoriteCount={tweet.favorite_count}
-            retweetCount={tweet.retweet_count}
-          />
-        ))}
-      </div>
-    );
-  } else {
-    return null;
-  }
+const styles = {
+  card: {
+    maxWidth: 345,
+  },
+  media: {
+    height: 140,
+  },
 };
 
-export default TweetList;
+const TweetList = ({ tweets }: ITweetListProps) => (
+  <Card>
+    <CardActionArea>
+      <CardContent>
+        <Typography gutterBottom variant="headline" component="h2">
+          {tweets[0].user.name} tweets ...
+        </Typography>
+        {tweets.map((tweet, index) => {
+          return (
+            !tweet.text.includes('RT ') && (
```

Comment:
These kind of logic and filtering belongs in the container component. Can you move it to `EventsContainer`? It might be convenient to use a filter instead of checking `tweet.text.includes('RT ')` for each tweet.

Reply:
And I have one more suggestions as wel; Can you also hide tweets that start with `@` (public mentions)?

Reply:
I do not mind either of these. Tweets are now filtered.